Week1task5/src/Array.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Arrays;
+public class Array {
+
+	public static void main(String[] args) {
+		//initialising an array
```

Comment:
Here please fix the spaces. The lines of code that follow have more  spaces before them than this line with the comment.